MdeModulePkg/MdeModulePkg.dec

```diff
@@ -1720,6 +1720,17 @@
   # @Prompt UFS device initial completion timoeout (us), default value is 600ms.
   gEfiMdeModulePkgTokenSpaceGuid.PcdUfsInitialCompletionTimeout|600000|UINT32|0x00000036
 
+  ## MU_CHANGE [BEGIN]
+  ## This PCD holds the background color of progress bar.
+  # Bit fields:
+  #   [31:24] - Reserved
+  #   [23:16] - Red
+  #   [15:8]  - Green
+  #   [7:0]   - Blue
+  # @Prompt Describes background color of progress bar.
+  gEfiMdeModulePkgTokenSpaceGuid.PcdProgressBarBackgroundColor|0x00808080|UINT32|0x0000010B
+  ## MU_CHANGE [END]

```

Comment:
Remove MU_CHANGE comment and expand description to state that the layout matches EFI_GRAPHICSS_OUTPUT_BLT_PIXEL_UNION.

Also, was a structured PCD considered so the DSC file that specifies the color could use the field names?

Reply:
Example of declaring a structured PCD in a DEC file:
 https://github.com/tianocore/edk2/blob/b6c4708c4d3470cfd9f465771a665510d3ad1a66/CryptoPkg/CryptoPkg.dec#L82

Example of setting field values in structured PCD in DSC file:

https://github.com/tianocore/edk2/blob/b6c4708c4d3470cfd9f465771a665510d3ad1a66/CryptoPkg/CryptoPkg.dsc#L197

Reply:
That was a mistake, thanks for pointing it out.  To be honest a structured PCD wasn't considered but that was due to the original author of the change not knowing that they exist.  It could be changed if being able to specify field names is valuable.

Reply:
given that structured PCDs are not very commonly used, this code has existed in downstream code bases for years, and dozens of products depend on it,  I would request that a superficial change like structured PCD is not required.

Reply:
Structured PCDs are extensively used in the CryptoPkg and are well supported.  It is easier from a platform integration perspective to have names for the Red, Green, Blue level in the DSC than a 32-bit hex value.